java/client/src/org/openqa/selenium/remote/http/netty/NettyClient.java

```diff
@@ -86,5 +90,14 @@ public HttpClient createClient(ClientConfig config) {
 
       return new NettyClient(new NettyHttpHandler(config, httpClient).with(config.filter()), NettyWebSocket.create(config, httpClient));
     }
+
+    @Override
+    public void cleanupIdleClients() {
+      try {
+        httpClient.close();
```

Comment:
The `httpClient` is a `static` field. In the case of a regular selenium test, closing this might be okay, but we also use this in the Grid, and in situations where more than one http client is being used. Surely this will close other `NettyClient` implementations in the same JVM? Or do I misunderstand something?